src/WCSimConstructRealisticPlacement.cc

```diff
@@ -654,8 +654,8 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructRealisticPlacement()
     // Since assembly volumes just expand relative to their center point the means
     // all nested volumes will also have the PMT againt the tyvek.
 
-    G4ThreeVector pmt_central_position = G4ThreeVector(config.InnerDetectorOuterRadius-59.62*mm,0.0,0.0);
-    G4ThreeVector mpmt_central_position = G4ThreeVector(config.InnerDetectorOuterRadius-59.62*mm,0.0,0.0);
+    G4ThreeVector pmt_central_position = G4ThreeVector(config.InnerDetectorOuterRadius-2*mm,0.0,0.0);
+    G4ThreeVector mpmt_central_position = G4ThreeVector(config.InnerDetectorOuterRadius-2*mm,0.0,0.0);
```

Comment:
```suggestion
    G4ThreeVector pmt_central_position = G4ThreeVector(config.InnerDetectorOuterRadius-59.62*mm,0.0,0.0);
    G4ThreeVector mpmt_central_position = G4ThreeVector(config.InnerDetectorOuterRadius-59.62*mm,0.0,0.0);
```
I'd just like to run the CI with this change reverted. There hasn't been any discussion as to why it was required. And I'd like to, if possible, not alter the ID as this will require fitqun tuning

Reply:
Hi Tom, the change to the ID PMT offset was to bring them flush to the wall.